### PR TITLE
Gh309 find series by catalog number

### PR DESCRIPTION
**Pull request by cssru**

### CODE REVIEW

**Comment by php-coder** [comment]
Why there is 2 commits? Did you update your master and rebase? Looks like you didn't.

File: src/main/java/ru/mystamps/web/dao/impl/JdbcSeriesDaoImpl.java [hunk 1]
```diff
@@ -211,63 +222,31 @@ public long countStampsOfCollection(Integer collectionId) {
...
-				Collections.singletonMap("michel_number", michelNumber),
-				Integer.class
-			);
-			return Optional.of(seriesId);
-			
-		} catch (EmptyResultDataAccessException ignored) {
-			return Optional.empty();
-		}
+	public List<Integer> findSeriesIdsByMichelNumberCode(String michelNumber) {
+		return jdbcTemplate.queryForList(findSeriesIdsByMichelNumberSql,
```
Comment (php-coder) [inline src/main/java/ru/mystamps/web/dao/impl/JdbcSeriesDaoImpl.java]: Move the argument to the new line, please.

File: src/main/webapp/WEB-INF/views/series/search_result.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	</div>
+	<div class="row">
+		<footer class="col-sm-12 text-right">
+			<i class="glyphicon glyphicon-envelope"></i>
+			<a href="mailto:slava.semushin@gmail.com" title="Write e-mail" th:href="|mailto:#{t_site_author_email}|" th:title="#{t_write_email}" th:text="#{t_site_author_name}">Slava Semushin</a>, 2009-2016
+		</footer>
+	</div>
+</div>
+
+<!-- Placed at the end of the document so the pages load faster -->
```
Comment (php-coder) [inline src/main/webapp/WEB-INF/views/series/search_result.html]: Форматирование.. :-(

File: src/main/java/ru/mystamps/web/service/SeriesServiceImpl.java [hunk 3]
```diff
@@ -201,41 +202,57 @@ public long countStampsOf(Integer collectionId) {
...
+	public List<SeriesInfoDto> findByMichelNumber(String michelNumberCode, String lang) {
 		Validate.isTrue(michelNumberCode != null, "Michel number code must be non null");
 		Validate.isTrue(!michelNumberCode.trim().isEmpty(), "Michel number code must be non empty");
 		
-		return seriesDao.findSeriesIdByMichelNumberCode(michelNumberCode);
+		List<Integer> seriesIds = seriesDao.findSeriesIdsByMichelNumberCode(michelNumberCode);
+		
+		return seriesIds.isEmpty()
+			? Collections.emptyList()
+			: seriesDao.findByIdsAsSeriesInfo(seriesIds, lang);
```
Comment (cssru) [inline src/main/java/ru/mystamps/web/service/SeriesServiceImpl.java]: И чего плохого? Красиво вроде бы ))
Checkstyle пропускает.

**Comment by php-coder** [comment]
Dupe for #319

**Pull request closed without merging**